src/test/java/com/github/iveina2021/automationexercises/SignUpPage.java

```diff
@@ -0,0 +1,132 @@
+package com.github.iveina2021.automationexercises;
+
+// page_url =  https://automationexercise.com/signup/
+
+import com.codeborne.selenide.SelenideElement;
+import org.openqa.selenium.support.FindBy;
+
+import static com.codeborne.selenide.Selenide.page;
+
+public class SignUpPage {
+    @FindBy(css = ".login-form > .title")
+    public SelenideElement enterAccountInformationLabel;
+
+    @FindBy(css = "input[value='Mr']")
+    public SelenideElement genderCheckboxMr;
+
+    @FindBy(css = "input[id='password']")
+    public SelenideElement passwordInput;
+
+    @FindBy(css = "select[id='days']")
+    public SelenideElement daysDropDown;
+
+    @FindBy(css = "select[id='days'] option[value='2']")
+    public SelenideElement getDay2DropDown;
+
+    @FindBy(css = "select[id='months']")
+    public SelenideElement monthsDropDown;
+
+    @FindBy(css = "select[id='months'] option[value='4']")
+    public SelenideElement getMonthApril;
+
+    @FindBy(css = "select[id='years']")
+    public SelenideElement yearsDropDown;
+
+    @FindBy(css = "option[value='2019']")
+    public SelenideElement getYear2019;
+
+    @FindBy(css = "input[id='newsletter']")
+    public SelenideElement newsletterCheckbox;
+
+    @FindBy(css = "input[id='optin']")
+    public SelenideElement receiveSpecialOffersCheckbox;
+
+    @FindBy(css = "div[class='form-group'] label")
```

Comment:
**suggestion**  this css selector is too common, and not related to dateBirthLabel.

So if the form contains multiple labels you will get multiple results for this selector. Can we rework it somehow?

Reply:
Don't changed CSS selector, but added for both scroll methods a check text of labels